src/main/java/bio/terra/service/resourcemanagement/exception/BufferServiceAPIException.java

```diff
@@ -0,0 +1,53 @@
+package bio.terra.service.resourcemanagement.exception;
+
+import bio.terra.buffer.client.ApiException;
+import bio.terra.common.exception.ErrorReportException;
+import java.util.Collections;
+import java.util.List;
+import org.springframework.http.HttpStatus;
+
+/** Wrapper exception for non-200 responses from calls to Buffer Service. */
+public class BufferServiceAPIException extends ErrorReportException {
```

Comment:
It doesn't look like all the exception constructors are used. Could we prune it so that only the used ones are defined?